components/charts/BarChart.tsx

```
@@ -29,6 +29,7 @@ type CartesianChartProps<
     ttY: SharedValue<number>;
   };
   barCount?: number;
+  formatXLabel?: (value: any) => string;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)